bitnami/thanos/templates/bucketweb/deployment.yaml

```diff
@@ -87,11 +88,27 @@ spec:
             - name: http
               containerPort: 8080
               protocol: TCP
-          {{- if .Values.bucketweb.livenessProbe }}
-          livenessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.bucketweb.livenessProbe "context" $) | nindent 12 }}
+          {{- if .Values.bucketweb.livenessProbe.enabled }}
+          livenessProbe: 
+            httpGet:
+              path: /-/healthy
+              port: http
+            initialDelaySeconds: {{ .Values.bucketweb.livenessProbe.initialDelaySeconds }}
+            timeoutSeconds: {{ .Values.bucketweb.livenessProbe.timeoutSeconds }}
+            periodSeconds: {{ .Values.bucketweb.livenessProbe.periodSeconds }}
+            successThreshold: {{ .Values.bucketweb.livenessProbe.successThreshold }}
+            failureThreshold: {{ .Values.bucketweb.livenessProbe.failureThreshold }}
```

Comment:
I would leave this and the readinessProbe using tplValues. It removes suplications and also makes the httpGet configurable

Reply:
@miguelaeh -> This is very opinionated in your team. As you can see here [https://github.com/bitnami/charts/pull/6461/files/5212d01466cf85125f78c40210389e7a7c04e686#r642593610](https://github.com/bitnami/charts/pull/6461/files/5212d01466cf85125f78c40210389e7a7c04e686#r642593610)

Reply:
@Cellebyte is right

@miguelaeh we should follow our standard and avoid using `tplValues` for probes (remember some discussions such as this one: https://github.com/bitnami/charts/pull/5048#discussion_r559451038)

Reply:
👍